ui/pages/settings/networks-tab/networks-list/networks-list.js

```diff
@@ -9,20 +16,73 @@ const NetworksList = ({
   networkDefaultedToProvider,
   selectedRpcUrl,
 }) => {
+  const t = useI18nContext();
+  const [searchedNetworks, setSearchedNetworks] = useState([]);
+  const [searchQuery, setSearchQuery] = useState('');
+  const searchedNetworksToRender =
+    searchedNetworks.length === 0 && searchQuery === ''
+      ? networksToRender
+      : searchedNetworks;
+  const searchedNetworksToRenderThatAreNotTestNetworks = searchedNetworksToRender.filter(
+    (network) => network.isATestNetwork === false,
+  );
+  const searchedNetworksToRenderThatAreTestNetworks = searchedNetworksToRender.filter(
+    (network) => network.isATestNetwork === true,
```

Comment:
Similarly, what about:

``` suggestion
    (network) => network.isATestNetwork,
```

Reply:
@mcmire You are write, I commited your suggestion. I remembered that I have problem with `lint` when I created this PR (a long time ago), and because of that I put `=== false` or `=== true`. But now it is working perfect, `lint` passed! Thank you for this suggestion!